app.js

```diff
@@ -4,8 +4,116 @@ const cancel = document.querySelector('.cancel');
+      featuredImage: './images/photo2dt.png',
+      desktopImage: './images/photo2dt.png',
+      technologies: ['html', 'css', 'javascript','ruby'],
+      linktoliveversion: 'https://kaylemba.github.io/MyPortfolio2/',
+      linktosource: 'https://github.com/KayLemba/MyPortfolio2',
+    },
+    {
+      key: 2,
+      name: 'FACEBOOK 360',
+      option: ['FACEBOOK', 'Full Stack Dev', '2015'],
+      description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent ',
+      featuredImage: './images/photo3dt.png',
+      desktopImage: './images/photo3dt.png',
+      technologies: ['html', 'css', 'javascript', 'ruby'],
+      linktoliveversion: 'https://kaylemba.github.io/MyPortfolio2/',
+      linktosource: 'https://github.com/KayLemba/MyPortfolio2',
+    },
+    {
+      key: 3,
+      name: 'Uber Navigation',
+      option: ['Uber', 'Lead Developer', '2018'],
+      description: ' Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+      featuredImage: './images/photo4dt.png',
+      desktopImage: './images/photo4dt.png',
+      technologies: ['html', 'css', 'javascript', 'git hub', 'ruby','Bootstrap'],
+      linktoliveversion: 'https://kaylemba.github.io/MyPortfolio2/',
+      linktosource: 'https://github.com/KayLemba/MyPortfolio2',
+    },
+  ];
+  const Openpopup = ({ 
+      name, description, featuredImage, desktopImage, technologies, linktoliveversion, linktosource, option
+  }) => () => {
+    let displayModal = '';
+    modal.classList.add('show-modal');
+    displayModal += `
+         <div class="modal-head">
+          <h3 class="modal-title">${name}</h3>
+         </div>
+         <ul class="card-detail-header-container">
+             <li class="card-detail canopy c-js">CANOPY</li>
+             <li class="card-detail option choice"><img src="images/bullets.png" alt="bullet"> Back End Dev</li>
+             <li class="card-detail option choice"><img src="images/bullets.png" alt="bullet"> 2015</li>
+            <li class="dcanopy2_face canopy-js">${option[0]}</li>
+            <li class="optiond2_full choice-javas"><img src="images/bullets.png" alt="bullet">${option[1]?option[1]:""}</li>
+            <li class="optiond2_full choice-javas"><img src="images/bullets.png" alt="bullet">${option[2]?option[2]:""}</li>
+          </ul>
+
+          <div class='modal-img-container' >
```

Comment:
- [ ] You're missing the Javascript linter. kindly update the linters.yml file check this [link](https://github.com/microverseinc/linters-config/tree/master/html-css-js/.github/workflows)

Well done on implementing the pop-up. It's responsive and it works as expected. However, there are some minor changes that you should do to make it look better.
![image](https://user-images.githubusercontent.com/43698511/132741517-a11056ea-8c6e-42d5-bc89-0a952a5e7fa3.png)

- [ ] I suggest adding some space between the bullets and the words to make it match the design.
- [ ] The images on desktop view look stretched. Kindly adjust their size.

Reply:
Done